backend/.env

```
@@ -0,0 +1,6 @@
+PORT=5001
+MONGO_URI=mongodb+srv://manhdung05072005:manhdung123123@streamify.lc49cum.mongodb.net/?retryWrites=true&w=majority&appName=streamify
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The MONGO_URI key should go before the PORT key

(UnorderedKey)